tests/test-sources/example-configurations/issues.nix

```diff
@@ -81,7 +81,7 @@
       nvim-tree = {
         enable = true;
         openOnSetup = true;
-        openOnTab = true;
+        tab.sync.open = true;
```

Comment:
This should be in a nvim-tree PR no ? or is the test currently broken ?

Reply:
Yes, I made a separate PR